frontend/src/concepts/pipelines/NoPipelineServer.tsx

```diff
@@ -18,16 +18,20 @@ const NoPipelineServer: React.FC<NoPipelineServerProps> = ({
 
   return (
     <EmptyDetailsView
-      title="Start by creating a pipeline server"
-      description="Standardize and automate machine learning workflows to enable you to further enhance and deploy your data science models."
+      title={installed ? 'Start by importing a pipeline' : 'Enable pipelines'}
+      description={`Pipelines are platforms for building and deploying portable and scalable machine-learning (ML) workflows. You can import a pipeline or create one in a workbench.${
+        installed
+          ? ' Before you can work with pipelines, you must first configure a pipeline server in your project.'
+          : ''
```

Comment:
I'm not sure how to properly configure a pipeline, so I'm not able to see both empty states.

I suspect the conditional rendering here is swapped, because for the **Enable pipelines** emptystate, I do not see the following statement which should should only display when a pipeline server has not been configured (i.e. the **Enable pipelines** empty state):
> Before you can work with pipelines, you must first configure a pipeline server in your project.

Maybe this is showing up for the **Start by importing a pipeline** empty state instead?

Reply:
As a side comment, I assume the variable `installed` is meant to reflect the status of a pipeline server being configured and not the presence of a pipeline operator on the cluster, but I could see future devs confusing this variable to mean the latter.